### ISSUE TITLE
ons-tab custom icon CSS class not displaying

### ISSUE THREAD

**Issue by CrankedCode** [OnsenUI/OnsenUI#2108]
I have my CSS class setup like so:
```
.orangeBottleIcon {
  background-image: url('images/bottle_orange.png');
  width: 100%;
  height: 100%;
}
```

HTML:
`<ons-tab page="shopsBrowse.html" id="browseTabShops" onclick="appFunc.loadShopsBrowseCache();" icon="orangeBottleIcon"></ons-tab>`

However it does not work.  Font-awesome and ionicons display fine.  In the ons-tab docmentation, it states "If you need to use your own icon, create a CSS class with background-image or any CSS properties and specify the name of your CSS class here.", however this does not appear to be the case.

Anything I'm missing here? Using Onsen v2.3.2

**Comment by frandiox** [comment]
@CrankedCode Hi, thanks for reporting! The `ons-icon` inside `ons-tab` was adding `fa-` prefix automatically to your class. This will be fixed in v2.5.0 :+1:

**Comment by frandiox** [comment]
@CrankedCode Sorry, I was wrong, this won't be fixed in 2.5.0. We will just remove those docs.
